lib/components/term.js

```diff
@@ -263,7 +264,10 @@ export default class Term extends React.PureComponent {
       });
     }
 
-    if (this.props.fontSize !== nextProps.fontSize || this.props.fontFamily !== nextProps.fontFamily) {
+    const fontSizeChanged = this.props.fontSize !== nextProps.fontSize;
+    const fontFamilyChanged = this.props.fontFamily !== nextProps.fontFamily;
+    const lineHeightChanged = this.props.lineHeight !== nextProps.lineHeight;
+    if (fontSizeChanged || fontFamilyChanged || lineHeightChanged) {
```

Comment:
I think that storing comparison result in temp vars is not necessary.
These comparisons don't need some explanation with explicit variable names. And without these temporary variables, comparisons will be done only if previous one is invalid.

Reply:
Updated.